blazor/datagrid/excel-like-filter.md

```diff
@@ -826,6 +826,14 @@ public class OrderData
 {% endhighlight %}
 {% endtabs %}
 
+## Add current selection to filter checkbox
+
+By default, the CheckBox filter can only filter the selected items. If filtering is done multiple times on the same column, the previously filtered values in the column will be cleared. Now, it is possible to retain those previous values by using the Add current selection to filter checkbox. This checkbox is displayed when data is searched in the search bar of the CheckBox filter.
```

Comment:
1) title should be --->  Add current selection to filter
2) This checkbox is displayed when data is searched in the search bar of the CheckBox or Excel filter type.  ---> Use this content, since you're using checkbox filter, it supports both filter types. 
3)Now, it is possible to retain those previous values by using the Add current selection to filter checkbox. ---> Here also modify the Add current selection to filter instead of Add current selection to filter checkbox.

Reply:
change content as per requirement.